### PR TITLE
GG host app: use scenes instead of UIScreen.main

### PR DESCRIPTION
**Pull request by axelmolch**
UIScreen.main is being deprecated.
This PR updates GG project to use scenes instead. 
Also changes target deployment iOS to 13.0.

### CODE REVIEW

**Comment by axelmolch** [comment]
built/executed on ipone 12, iOS 16.0.3

**Review by davidahouse — APPROVED** [review]
👍

File: platform/apple/apps/host/GoldenGate-Host-iOS/SceneDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+//  SceneDelegate.swift
+//  GoldenGateHost
+//
+//  Created by Axel Molchanov on 1/10/23.
+//  Copyright © 2023 Fitbit. All rights reserved.
+//
+
+import UIKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
```
Comment (emixb) [inline platform/apple/apps/host/GoldenGate-Host-iOS/SceneDelegate.swift:11]: nit: final class

**Review by emixb — APPROVED** [review]

**Review by barbibulle — APPROVED** [review]

**Comment by axelmolch** [comment]
for actual diff take `axel_m1_migration_0105_940` for base

**Pull request merged**